### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character '\u2552' in position 0: ordinal not in range(128)

### ISSUE THREAD

**Issue by chuanmingliu** [jhfjhfj1/autokeras#195]
### Bug Description

Hi, everyone! When I try to use autokeras fit some data and label, I encounter such errors as:
 
"Using TensorFlow backend.

Initializing search.
Initialization finished.

Traceback (most recent call last):
  File "ak.py", line 14, in <module>
    clf.fit(x_train, y_train, time_limit=12*60*60)
  File "/usr/local/lib/python3.6/dist-packages/autokeras/image_supervised.py", line 239, in fit
    run_searcher_once(train_data, test_data, self.path, int(time_remain))
  File "/usr/local/lib/python3.6/dist-packages/autokeras/image_supervised.py", line 40, in run_searcher_once
    searcher.search(train_data, test_data, timeout)
  File "/usr/local/lib/python3.6/dist-packages/autokeras/search.py", line 172, in search
    print('\u2552' + '=' * 46 + '\u2555')
UnicodeEncodeError: 'ascii' codec can't encode character '\u2552' in position 0: ordinal not in range(128)"

Does anyone have experience in fixing this?

Here is my code:

```
from autokeras.image_supervised import ImageClassifier
import numpy as np

if __name__ == '__main__':
    x_train = np.load('x_train.npy')
    y_train = np.load('y_train.npy')
    x_test = np.load('x_test.npy')
    y_test = np.load('y_test.npy')
    x_train = x_train.reshape((-1, 32, 32, 3))
    y_train.reshape((-1, 1))
    x_test = x_test.reshape((-1, 32, 32, 3))
    y_test = y_test.reshape((-1, 1))
    clf = ImageClassifier(verbose=True)
    clf.fit(x_train, y_train, time_limit=12*60*60)
    clf.final_fit(x_train, y_train, x_test, y_test, retrain=True)
    pred = clf.evaluate(x_test, y_test)
    print(pred)
```

Thanks in advance!
### Reproducing Steps
  * Step 1: ...
  * Step 2: ...

### Setup Details
Include the details about the versions of:
 - OS type and version: Ubuntu 16.04
 - Python: 3.6
 - autokeras:  0.2.14
 - scikit-learn:
 - numpy:
 - keras: 2.2.2
 - scipy:
 - tensorflow:  1.10.0
 - pytorch:

**Comment by jhfjhfj1** [comment]
@chuanmingliu Thanks for trying out.
Are you able to run the mnist example?

**Comment by chuanmingliu** [comment]
Exactly the same as:
**"root@91686962c4c1:/data/autokeras# python minist_example.py
Using TensorFlow backend.
Downloading data from https://s3.amazonaws.com/img-datasets/mnist.npz
11493376/11490434 [==============================] - 8s 1us/step

Initializing search.
Initialization finished.


Traceback (most recent call last):
  File "minist_example.py", line 10, in <module>
    clf.fit(x_train, y_train, time_limit=12 * 60 * 60)
  File "/usr/local/lib/python3.6/dist-packages/autokeras/image_supervised.py", line 239, in fit
    run_searcher_once(train_data, test_data, self.path, int(time_remain))
  File "/usr/local/lib/python3.6/dist-packages/autokeras/image_supervised.py", line 40, in run_searcher_once
    searcher.search(train_data, test_data, timeout)
  File "/usr/local/lib/python3.6/dist-packages/autokeras/search.py", line 172, in search
    print('\u2552' + '=' * 46 + '\u2555')
UnicodeEncodeError: 'ascii' codec can't encode character '\u2552' in position 0: ordinal not in range(128)"**

**Comment by fliessen** [comment]
have same problem with docker.
and i installed it with source code.

**Comment by eagleoflqj** [comment]
try this command:
PYTHONIOENCODING=utf-8 python minist_example.py

**Comment by chuanmingliu** [comment]
> try this command:
> PYTHONIOENCODING=utf-8 python minist_example.py

Thanks for your reply!  Yes, this is the right solution. But I encounter another problem with respect to "Bus error" can be solved by [#118](https://github.com/jhfjhfj1/autokeras/issues/118)

**Comment by chuanmingliu** [comment]
This problem has been well solved and I think this issue can be closed. Thanks again @jhfjhfj1 @eagleoflqj
